scania-digital-design-system/sdds
# fix(textarea/textfield): handle null values textfield/textarea

**Describe pull-request**  
Added null checks to allow for null values on our inputfields. With this the component will render even though it's value might be null. 

**Solving issue**  
Fixes: #526 [AB#2445](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2445)

**How to test**  
1. Checkout the branch and cd to textarea/textfield component.
2. Change de default value of the components to null
3. Check that the components render without issues.

**Screenshots** 
-

**Additional context**  
-
